src/lwc/geGiftEntryFormApp/geGiftEntryFormApp.js

```diff
@@ -351,6 +385,23 @@ export default class GeGiftEntryFormApp extends NavigationMixin(LightningElement
         }));
     }
 
+    displayExpiredAuthorizationWarningModal() {
```

Comment:
@voduyemi - I went with this approach for displaying the modal - not just building the modal detail but also firing the togglemodal event within this method. Let me know if you would prefer to see this broken down with this function just returning the modal detail structure!

Reply:
This works for me. Thanks!